web/src/features/charts/bar-breakdown/ProductionSourceLegend.tsx

```diff
@@ -12,17 +12,13 @@ export default function ProductionSourceLegend({
   return (
     <g className="pointer-events-none">
       <rect
-        transform={`translate(${LABEL_MAX_WIDTH - 1.5 * PADDING_Y - 10}, 0)`}
+        transform={`translate(0, 0)`}
```

Comment:
Unless I am mistaken I don't think `translate(0,0)` does anything so it can be removed?

Reply:
True